Makefile

```diff
@@ -1,26 +1,66 @@
+.PHONY: help venv conda docker docstyle format style black test lint check
+.DEFAULT_GOAL = help
+
+PYTHON = python
+PIP = pip
+CONDA = conda
+SHELL = bash
+
+help:
+	@printf "Usage:\n"
+	@grep -E '^[a-zA-Z_-]+:.*?# .*$$' $(MAKEFILE_LIST) | awk 'BEGIN {FS = ":.*?# "}; {printf "\033[1;34mmake %-10s\033[0m%s\n", $$1, $$2}'
+
+conda:  # Set up a conda environment for development.
+	@printf "Creating conda environment...\n"
+	${CONDA} create --yes --name env python=3.6
+	( \
+	${CONDA} activate env; \
+	${PIP} install -U pip; \
+	${PIP} install -r requirements.txt; \
+	${PIP} install -r requirements-dev.txt; \
+	${CONDA} deactivate; \
+	)
+	@printf "\n\nConda environment created! \033[1;34mRun \`conda activate env\` to activate it.\033[0m\n\n\n"
+
+venv:  # Set up a Python virtual environment for development.
+	@printf "Creating Python virtual environment...\n"
+	rm -rf venv
+	${PYTHON} -m venv venv
```

Comment:
Same -> `pymc4`.

Reply:
Python venvs make another directory to store the dependencies. It can't be called `pymc4` because there's already a `pymc4`! I could call it `pymc4-venv`.